.circleci/config.yml

```diff
@@ -74,12 +74,19 @@ jobs:
             sudo apt-get update
             sudo apt-get install python3-pip python3-dev
             sudo pip3 install awscli
+      - run:
+          name: Install Sentry CLI
+          command: |
+            npm install @sentry/cli
```

Comment:
Since sentry is now specific to our app i think we should _not_ install it globally. Instead can we include it in GA4 and/or do `npx sentry-cli ...`?

Reply:
but the release is not – it's specific to this whole repository – and the sentry project is as well.

Reply:
I'm picturing that we do this globally as soon as we want to add the next app and then each app is just responsible for their own source and source map files part. open to thoughts about this

Reply:
my opinion here is that we should keep the installation of the dependency coupled to the place where it's used. so if we have a script in GA4 then it _should_ be declared in GA4 and not implicitly depend on an outside context. inverse would be true too, why install a global dependency when it's only used in one place?

Reply:
yeah fair enough. we can rearrange it later on then. i'll adjust – thanks for the input